### PR TITLE
Update scheduled-changes.md

### PR DESCRIPTION
**Pull request by umahasit**
Rule update for 100038

### CODE REVIEW

**Comment by KimJ15** [comment]
@pedrosousa would you mind taking a look at this PR?

File: products/waf/src/content/change-log/scheduled-changes.md [hunk 1]
```diff
@@ -31,6 +29,29 @@ order: 1
...
+                2021-04-19
+            </td>
+            <td>
+                Cloudflare Specials
+            </td>
+            <td>
+                100038
+            </td>
+            <td>
+                Rule Improvement
```
Comment (migueldemoura) [inline products/waf/src/content/change-log/scheduled-changes.md]: Could we be more specific here? Did we improve detection, reduce false positives or performance?

Comment (umahasit) [reply products/waf/src/content/change-log/scheduled-changes.md]: "Improve Information Disclosure Detection" would summarise it well. Detection or Coverage works

Comment (umahasit) [reply products/waf/src/content/change-log/scheduled-changes.md]: Changed to Improve Information Disclosure Coverage

**Review by migueldemoura — APPROVED** [review]

**Pull request merged**